### ISSUE TITLE
Releases to package repositories

### ISSUE THREAD

**Issue by MatthewFlamm** [OPM/pyopmspe11#87]
It would be easier to use for users if they could `pip install pyopmspe11` from the PyPI repository.  This also makes it easier to get a specific released version compared to the git based options [related to #86].  Other options to consider are `conda` (related to #79) through `conda-forge`.

Related to https://github.com/openjournals/joss-reviews/issues/7357

**Comment by daavid00** [comment]
Thanks @MatthewFlamm for this suggestion. After merging https://github.com/OPM/pyopmspe11/pull/95, which was also your feedback and very useful, then now we have extended the documentation in the Python installation to explain how one can install a specific release of this repository (https://opm.github.io/pyopmspe11/installation.html#python-package). We also mentioned that package managers such as Anaconda are not tested. Then, @totto82 and I we believe that with the provided documentation the installation of this repository could be achieved, and it is not planned to make package versions of this repository for now.

**Comment by MatthewFlamm** [comment]
I don't think this issue is necessarily closed, but the changes are sufficient for purposes of https://github.com/openjournals/joss-reviews/issues/7357.
